src/content/time-tracking.mdx

```diff
@@ -0,0 +1,65 @@
+
+Warning: please **don’t track project tasks without JIRA usage**. It’s really important to do it this way because of not just easy linking. JIRA should has updated timers in tasks to verify estimated time vs spend time. It’s possible only this way.
+
+### Project’s stuff
+Each project has a list of categories in Harvest, but only a development (or similar) is what you can track by JIRA connection. **All project things, which are not tasks you should track by Harvest, choosing the related category (meeting, code review, estimation etc.) and adding a description.** The last thing is really important to always be sure what meeting/estimation took how much time. Descripition of code review time is not obligatory.
+
+OMG, the expected project is not on the list in Harvest, what should I do?
+If you want to do a code review (or something else) for somebody from not-your project, please ask your PM to enable this project for you. It’s important to ask before starting doing this, because your PM should be informed about it and finally decide is it OK.
+
+### Other tasks
+Luckily our working day is not only a project stuff. Each developer is added to „Dev Team Internal” project, which is created to track your time spent on developing monthly dev team tasks, learning, meetings (including 1on1s). Remember about a description to this kind of tasks, to make it clear what a dev task you are doing, what do you do while learning, what a meeting you’re participating etc.
+
+
+## Tracking good manners
+
+### Adding a description
+As mentioned a few times earlier, it’s an expected practice to add descriptions to your tracked positions. **Remember that you don’t need to describe them in details. Try to use a short definition of described task.** E.g. daily, planning, refinement (for meetings), Accessibility/A11y, TypeScript generics, Unit tests (for learning), a copy-pasted name of your dev task. The main goal of a description is to easy&fast define what it is, to easy&fast inform a PM about it.
+
+### Regularity - keep your PM always informed
+It’s not obvious but **your timesheet should always be up to date**. It’s not good to fill a timesheed at the end of a day or a week, because it’s impossible to fill it accurately this way and it’s hard to prevent extra hours you spend on a task. **The best and expected way of tracking time is to keep a timer always running, when you’re working.**
+It’s not only for PMs, but for you as developer too, because when it will be visible in a timesheet, that you’re working on an important task now, then nobody will disturb you :)
+
+### Overtime
+It’s important to **control your timesheet by yourself to prevent extra hours.** It means you should control your estimated vs spent time on a task, your break time (don’t go beyond a limit, but respect a daily rest) and your weekly total hours. Remember to control it always, even if it’s a release time or something.
```

Comment:
```suggestion
It’s important to **control your timesheet by yourself to prevent extra hours.** It means you should control your estimated vs spent time on a task, your break time (don’t go beyond a limit, but respect a daily rest) and your weekly total hours.
```

Let's not make it too 🌶️